src/components/search/index.tsx

```diff
@@ -284,6 +285,7 @@ const Search: FunctionComponent<SearchProps> = ({
                     )}
                   <ScrollElement name="hits" />
                   <Stats searchQuery={searchState.query} />
+                  {/* HIDE_DUPLICATES: the Hits component renders the Algolia Hits component which connects to an Algolia store that is tied in to any changes in the results that come from altering the search and/or facets of the search. So, the results that we show on the search page are deeply tied in to what Algolia returns. Because Instructors are one of the facets that Algolia searches against, we are going to have to let Algolia in on the action. */}
```

Comment:
I assume the Algolia data updates are driven from Rails? If so, then we'd need to add that boolean to skip indexing the unlisted instructors? Or is it safe to simply let both instructors get indexed since it's the same content?

Reply:
I'd be more inclined to tag the unlisted instructor over adding a new column to the database since this is a 1 off, but whatever makes sense.

Here's how the instructor relationship works. It's a **facet** of the content models:

https://www.loom.com/share/488eb44b08c74e4084cf3911274c6809

https://github.com/eggheadio/egghead-rails/blob/1d4e101d16c3062fd32025baf6dd8aa9b4fc30a1/app/models/playlist.rb#L222-L242

https://github.com/eggheadio/egghead-rails/blob/686ed03fcd70df56561da01a3ff99702d99c0114/app/models/lesson.rb#L10-L30

Reply:
There are no tags directly on the instructor are there? The lessons/playlists aren't what we need to hide. I'm all for tags on the instructor if we can do that. (Although, if the tags are displayed anywhere, a random `unlisted` tag would be awkward.)